dsc/cli.py

```diff
@@ -29,9 +29,24 @@
 @click.option(
     "-b",
     "--batch-id",
-    help="The S3 prefix for the batch of DSpace submissions",
+    help="The S3 prefix for the batch of DSpace submission files",
     required=True,
 )
+@click.option(
+    "-s",
+    "--s3-bucket",
+    help="The S3 bucket containing the DSpace submission files",
+)
+@click.option(
+    "-o",
+    "--output-queue",
+    help="The SQS output queue for the DSS result messages",
+)
+@click.option(
+    "-e",
+    "--email-recipients",
+    help="The recipients of the submission results email",
+)
```

Comment:
Feel free to disregard if not true, but could we update the docstring for these to indicate passing these will _override_ possible defaults in the Workflow definition itself?